### ISSUE TITLE
Merging strategy for new release versions (CMIP6-compatible and INES interim release)

### ISSUE THREAD

**Issue by jmaerz** [NorESMhub/BLOM#289]
Hi @JorgSchwinger  and @TomasTorsvik ,

PR related in preparation to https://github.com/NorESMhub/NorESM/issues/466:

Required for **CMIP6-compatible** final version:
- #281 
- #291 
- #290
- potentially #233 
- #294
- #297
- #299 
- #300
- #301
- #303
- #305
- #307
- #308
- #309 (currently expected as the final and last CMIP6-compatible version from the HAMOCC side)
- #310 tagged release of CMIP6 compatible BLOM/HAMOCC

(Upcoming) PR for getting iHAMOCC ready for **NorESM2.1** version (PRs are named, once they are opened):
- #311 Merge `master` into `beyond-CMIP6`  & code cosmetics on `beyond-CMIP6`
- #314 Some minor declaration, etc. issues
- #315 Merge `beyond-CMIP6` (incl. bug-fixes) back into `master` - making `master` ready for milestone NorESM2.1
**From [cca8d11](https://github.com/NorESMhub/BLOM/commit/cca8d118e638d38e14eb860c60cfb49579b87728) onward, `master` is not backward compatible any longer for iHAMOCC** and ready for NorESM2.1.

(Upcoming) PR for getting iHAMOCC ready for **NorESM2.x** version (PRs are named, once they are opened):
- #269 
- potentially add preformed silicate tracer

**Comment by jmaerz** [comment]
**Notes for merging `master` into `beyond-CMIP6`:**
Check fixes formerly carried out: 
- DONE burial
- DONE OMP in sinking
- DONE alkalinity, etc. in sediment

Further changes to apply, once `master` is merged into `beyond-CMIP6`:
- DONE: `wpoc`, `wopal`, etc. as `protected` (via #307) 
- DONE adapt OMP settings in sinking
- #314 `bifr13`, `bifr14`

Changing opal dissolution rate in sediment will be carried out for a later tagged NorESM version
